gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -180,9 +205,10 @@ def preprocess(self, img):
     def reset(self):
         print('Resetting environment and starting new episode')
         self.controller.reset(self.scene_id)
-        self.event = self.controller.step(dict(action='Initialize', gridSize=0.25,
-                                               renderDepthImage=True, renderClassImage=True,
-                                               renderObjectImage=True))
+        self.event = self.controller.step(dict(action='Initialize', gridSize=self.gridSize,
```

Comment:
We need to merge Rainbow and then pull from this branch to update the rendering and maybe some other things. Let's do one thing after the other

Reply:
it's fine to not have the rendering